### PR TITLE
bugfix(TDI-38543) : When check 'De-duplicate' the two other options s…

### PR DESCRIPTION
**Pull request by undx**
…hould be hidden (REST API)

**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-38543

**What is the new behavior?**

Fixed.

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [X] The code coverage on new code >75%
- [X] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/121/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/121/)

File: components/components-marketo/src/main/java/org/talend/components/marketo/tmarketooutput/TMarketoOutputProperties.java [hunk 1]
```diff
@@ -208,9 +208,14 @@ public void refreshLayout(Form form) {
                 switch (outputOperation.getValue()) {
                 case syncLead:
                 case syncMultipleLeads:
-                    form.getWidget(operationType.getName()).setVisible(true);
-                    form.getWidget(lookupField.getName()).setVisible(true);
                     form.getWidget(deDupeEnabled.getName()).setVisible(true);
+                    if (deDupeEnabled.getValue()) {
```
Comment (jzhao-talend) [inline components/components-marketo/src/main/java/org/talend/components/marketo/tmarketooutput/TMarketoOutputProperties.java]: How about use this kind of code:
`form.getWidget(operationType.getName()).setHidden(deDupeEnabled.getValue());`

Comment (undx) [inline components/components-marketo/src/main/java/org/talend/components/marketo/tmarketooutput/TMarketoOutputProperties.java]: √

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/124/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/124/)

**Pull request merged**